app/javascript/pages/getAssistance/document-checklist.tsx

```diff
@@ -79,36 +83,52 @@ const DocumentChecklist = () => {
       ),
     },
     {
-      title: t("documentChecklist.assistedHousingTitle"),
-      expanded: (
-        <div className="space-y-2.5">
-          <p>{renderInlineMarkup(t("documentChecklist.assistedHousingDoc1"))}</p>
-          <p>{t("documentChecklist.assistedHousingDoc2")}</p>
-        </div>
-      ),
-    },
-    {
-      title: t("documentChecklist.rentBurdenTitle"),
+      id: PREFERENCES_IDS.assistedHousing,
+      title: t("listings.lotteryPreference.Rent Burdened / Assisted Housing Preference.title"),
       expanded: (
         <div className="space-y-2.5">
-          <p>{renderInlineMarkup(t("documentChecklist.rentBurdenDoc1"))}</p>
-          <p>{renderInlineMarkup(t("documentChecklist.rentBurdenDoc2"), "<ol><li>")}</p>
-          <ul className="list-disc ml-7">
-            <li>{t("label.proof.moneyOrder")}</li>
-            <li>{t("label.proof.cancelledCheck")}</li>
-            <li>{t("label.proof.debitFromBank")}</li>
-            <li>{t("label.proof.onlinePayment")}</li>
-          </ul>
+          <p>{renderInlineMarkup(t("documentChecklist.ociiSponsored"))}</p>
+          <p>{t("documentChecklist.twoWaysDesc")}</p>
+          <ol className="list-decimal list-inside ml-2">
+            <li>
+              <p className="pb-2.5">
+                {renderInlineMarkup(t("documentChecklist.assistedHousingDesc"))}
+              </p>
+              <ul className="list-disc list-inside ml-2">
+                <li>
+                  <p className="pb-2.5">{t("label.proof.leaseAgreement")}</p>
+                </li>
+              </ul>
+            </li>
+            <li>
```

Comment:
Also seeing the line issue here
<img width="1080" alt="Screenshot 2023-05-23 at 9 20 55 PM" src="https://github.com/SFDigitalServices/sf-dahlia-web/assets/131277283/e85c1b88-67dc-439f-b89c-5c3263a63708">